src/components/UserProfile.re

```diff
@@ -18,7 +18,7 @@ module Token = {
     let clearAndPush = RootProvider.useClearNonUrlStateAndPushRoute();
     let image = Animal.useAvatar(tokenId);
 
-    <div className=Css.(style([width(vh(12.))]))>
+    <div className=Css.(style([width(vh(12.)), cursor(`pointer)]))>
```

Comment:
Makes the animals on both the user and artist profiles display that they are clickable.